app/components/public/speaker-item.js

```diff
@@ -1,7 +1,26 @@
 import Ember from 'ember';
 
-const { Component } = Ember;
+const { Component, computed } = Ember;
 
 export default Component.extend({
-  classNames: ['column', 'four', 'wide']
+  classNameBindings: ['style'],
+
+  style: computed(function() {
+    const isFeatured = this.get('isFeatured');
+    if (isFeatured) {
+      return 'column eight wide';
+    } else {
+      return 'column four wide';
+    }
+  }),
+  didInsertElement() {
+    const isFeatured = this.get('isFeatured');
+    if (isFeatured) {
+      this.$('.speaker-item').addClass(' ui grid stackable');
```

Comment:
Okay so while computing the `featuredSpeakers` we will add a `isFeatured` attribute to the list and the list of `speakers` should not have `isFeatured` attribute. We are using `isFeatured` to distinguish featured speakers. So to avoid the above situation we should not have `isFeatured` in the `speakers` list.